From b7f85bc0d54bcd7fd86168cfcd4e039249e20c3d Mon Sep 17 00:00:00 2001
From: Andreas Oberritter <obi@linuxtv.org>
Date: Mon, 8 Aug 2011 14:29:50 +0000
Subject: [PATCH 1/3] DVB: Add SYS_TURBO for north american turbo code FEC
Cc: user.vdr@gmail.com,
    alannisota@gmail.com

- Allow to distinguish turbo code from DVB-S

Signed-off-by: Andreas Oberritter <obi@linuxtv.org>
---
 Documentation/DocBook/media/dvb/dvbproperty.xml |    1 +
 include/linux/dvb/frontend.h                    |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/linux/include/linux/dvb/frontend.h b/linux/include/linux/dvb/frontend.h
index 36a3ed6..1b1094c 100644
--- a/linux/include/linux/dvb/frontend.h
+++ b/linux/include/linux/dvb/frontend.h
@@ -349,6 +349,7 @@ typedef enum fe_delivery_system {
 	SYS_CMMB,
 	SYS_DAB,
 	SYS_DVBT2,
+	SYS_TURBO,
 } fe_delivery_system_t;
 
 struct dtv_cmds_h {
diff --git a/linux/include/linux/dvb/version.h b/linux/include/linux/dvb/version.h
--- a/linux/include/linux/dvb/version.h
+++ b/linux/include/linux/dvb/version.h
@@ -24,6 +24,6 @@
 #define _DVBVERSION_H_
 
 #define DVB_API_VERSION 5
-#define DVB_API_VERSION_MINOR 3
+#define DVB_API_VERSION_MINOR 4
 
 #endif /*_DVBVERSION_H_*/
-- 
1.7.2.5

